python_tests/test_csv_download.py

```diff
@@ -0,0 +1,123 @@
+from typing import Any
+
+import optuna
+from optuna.trial import TrialState
+from optuna_dashboard._app import create_app
+import pytest
+
+from .wsgi_client import send_request
+
+
+def test_download_csv_no_trial() -> None:
+    def objective(trial: optuna.Trial) -> float:
+        x = trial.suggest_float("x", -100, 100)
+        y = trial.suggest_categorical("y", [-1, 0, 1])
+        return x**2 + y
+
+    storage = optuna.storages.InMemoryStorage()
+    study = optuna.create_study(storage=storage)
+    study.optimize(objective, n_trials=0)
+    app = create_app(storage)
+    status, _, body = send_request(

```

Comment:
`body` is actually not used anywhere in all the tests, but should we also test it or shall we just make it `_` and ignore it?

```suggestion
    status, _, _ = send_request(
```